samples/azure/custom-partitioning-asb/ASB_7/Subscriber/Program.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+
+class Program
+{
+    static void Main()
+    {
+        MainAsync().GetAwaiter().GetResult();
+    }
+
+    static async Task MainAsync()
+    {
+        Console.Title = "Samples.ASB.Partitioning.Subscriber";
+        Console.WriteLine("Which connectionstring shall I subscribe to [1, 2]:");
```

Comment:
As per conversation, would be better to have two Subscriber projects, one per namespace w/o asking for user input.